tests/storage/storage.go

```diff
@@ -92,10 +93,18 @@ var _ = Describe(SIG("Storage", func() {
 
 	BeforeEach(func() {
 		virtClient = kubevirt.Client()
+		bindingMode := storagev1.VolumeBindingImmediate
+		sc := "manual"
+		libstorage.CreateStorageClass(sc, &bindingMode)
+
 		libstorage.CreateHostPathPv("alpine-host-path", testsuite.GetTestNamespace(nil), testsuite.HostPathAlpine)
 		libstorage.CreateHostPathPVC("alpine-host-path", testsuite.GetTestNamespace(nil), "1Gi")
 	})
 
+	AfterEach(func() {
+		libstorage.DeleteStorageClass("manual")
```

Comment:
Seems a bit hacky to hardcode the sc name here. I believe it's better to define it as a const or variable in the `Describe` scope and then use it during both CreateStorageClass and DeleteStorageClass.